.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "5.0"
```

Comment:
Isn't node-steam only supported for >=4.2 at the moment? Seems counter-intuitive to test versions not supported by our dependencies..